server/sonar-server/src/main/java/org/sonar/server/platform/monitoring/SonarQubeMonitor.java

```diff
@@ -100,7 +128,9 @@ public String name() {
     Map<String, Object> attributes = new LinkedHashMap<>();
     attributes.put("Server ID", getServerId());
     attributes.put("Version", getVersion());
-    attributes.put("External User Authentication", getExternalUserAuthentication());
+    addIfNotNull("External User Authentication", getExternalUserAuthentication(), attributes);
+    addIfNotEmpty("Identity Providers", getEnableIdentityProviders(), attributes);
+    addIfNotEmpty("Identity Providers allowing users to signup", getAllowsToSignUpEnabledIdentityProviders(), attributes);
```

Comment:
if my understanding is correct, it's the identity providers with which we can signup on SonarQube. I find the wording a little bit weird. Maybe check with our English guru?